pynicotine/gtkgui/ui/settings/nowplaying.ui

```diff
@@ -49,6 +49,14 @@
                 <signal name="toggled" handler="update_now_playing_info" swapped="no"/>
               </object>
             </child>
+            <child>
+              <object class="GtkRadioButton" id="NP_listenbrainz">
```

Comment:
While this is a good suggestion for a data provider, many other providers may be desired in the future. We should provide a hook for /plugins/ so this can be facilitated without needing to modify the core.